db/listener_test.cc

```diff
@@ -1183,6 +1188,32 @@ TEST_F(EventListenerTest, OnBlobFileOperationTest) {
   }
 }
 
+TEST_F(EventListenerTest, OnWALOperationTest) {
```

Comment:
Nit: in this test, we can test reading from both the MANIFEST file and WAL file. Therefore, we can rename the test to something like "ReadManifestAndWALOnRecovery".